index.bs

```diff
@@ -235,8 +252,26 @@ To <dfn>sanitize an attribute</dfn> named |attr|, run these steps:
 6. if |config| has a non-empty [=attribute allow list=] and |name| is not in |config|'s [=attribute allow list=] return 'drop'
 7. return 'keep'
 
+To <dfn>create a document fragment</dfn>
+named |fragment| from a Sanitizer |input|, run these steps:
+
+1. Switch based on |input|'s type:
+  1. if |input| is of type {{DocumentFragment}}, then:
+    1. let |node| refer to |input|.
+  2. if |input| is of type {{Document}}, then:
+    1. let |node| refer to |input|'s `documentElement`.
+  3. if |input| is of type `DOMString`, then:
+    1. let |node| be the result of the {{parseFromString}} algorithm
+        with |input| as first parameter (`string`),
+        and `"text/html"` as second parameter (`type`).
+2. let `f` be the result of {{createDocumentFragment}}.
+3. return the result of `f.cloneNode(node, true)`.
```

Comment:
I see `node . cloneNode([deep = false]) ` in spec, but you seem to be using `targetNode.cloneNode(sourceNode, true)` - What am I missing?

Reply:
I think I confused myself with importNode and cloneNode. But importNode is Document-only.

 think I now formalized this with correct references.